qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -30,22 +30,48 @@ def _resume_scrub(self, expected):
         self.assertEqual(res['return_code'], expected)
     def _get_scrub_status(self):
         return self.fs.rank_tell(["scrub", "status"])
-    def _check_task_status(self, expected_status):
-        task_status = self.fs.get_task_status("scrub status")
-        active = self.fs.get_active_names()
-        log.debug("current active={0}".format(active))
-        self.assertTrue(task_status[active[0]].startswith(expected_status))
+    def _check_task_status(self, expected_status, timo=120):
+        check = 0
+        while check < timo:
```

Comment:
check out `safe_while` elsewhere in the qa suite / teuthology

Reply:
ACK -- nice to know